### PR TITLE
fix(workflow): Show error message when roles list is invalid

### PR DESCRIPTION
**Pull request by billyvg**
+ log data when this happens.

Fixes JAVASCRIPT-2MJ

### CODE REVIEW

**Comment by billyvg** [comment]
Not sure what more to do here if roles list doesn't return from api call (it should always list roles)

File: src/sentry/static/sentry/app/views/inviteMember/inviteMember.jsx [hunk 1]
```diff
@@ -39,15 +39,31 @@ const InviteMember = React.createClass({
     let {slug} = this.getOrganization();
     this.api.request(`/organizations/${slug}/members/me/`, {
       method: 'GET',
-      success: ({roles}) => {
-        this.setState({roleList: roles, loading: false});
-        if (!roles || roles.filter(({allowed}) => allowed).length === 0) {
+      success: resp => {
+        let {roles} = resp || {};
+
+        if (roles.filter(({allowed}) => allowed).length === 0) {
```
Comment (MaxBittker) [inline src/sentry/static/sentry/app/views/inviteMember/inviteMember.jsx]: if roles is undefined, would this throw and not get to the capture message?

Comment (billyvg) [reply src/sentry/static/sentry/app/views/inviteMember/inviteMember.jsx]: Ah yeah sorry

**Review by MaxBittker — APPROVED** [review]
hopefully this will make the problem clear! thanks

( i'm so glad [] is truthy in js )

**Pull request merged**